data/maps.tsx

```diff
@@ -137,7 +139,47 @@ const IDforCardBonus: NumberMap = {
         72: "+{%_Total_Dungeon_Dmg",
         73: "+{%_Dungeon_MP_regen",
         74: "+{%_Dungeon_Drop_Rate",
-    }
+        75: "+{%_Catching_Speed",
+        76: "+{%_Breeding_EXP_(Passive)",
+        77: "+{%_Kitchen_Speed_(Passive)",
+        78: "+{_Star_Talent_Pts_(Passive)",
+        79: "+{%_Lab_EXP_gain",
+        80: "+{%_Multikill_per_tier",
+        81: "+{%_Critical_Chance_(Passive)",
+        82: "+{%_All_Stat",
+        83: "+{px_Line_Width_(Passive)",
+        84: "+{%_All_Skill_Efficiency",
+        85: "+{%_Cooking_EXP_gain",
+        86: "+{%_Dungeon_Move_Speed",
+        87: "+{%_Sailing_Speed_(Passive)",
+        88: "+{%_Skill_EXP_(Passive)",
+        89: "+{%_Divinity_EXP_(Passive)",
+        90: "+{%_Money_from_mobs_(Passive)",
+        91: "+{_Weapon_Power_(Passive)",
+        92: "+{%_Sailing_Speed_(Passive)",
+        93: "+{%_Skill_Efficncy_(Passive)",
+        94: "+{%_Class_EXP_(Passive)",
+        95: "+{%_All_Skill_EXP_(Passive)"
+}
+
+const SkillsforIDCardPassiveBonus: Record<SkillsIndex, number[]> = {
```

Comment:
I like this approach, much cleaner.

You wouldn't know this but generally speaking `maps.tsx` is a legacy piece of code that I barely ever touch so if possible I'd rather we move this map to somewhere more appropriate (either `rift.tsx` or `cards.tsx`)

Reply:
As it's related to Cards and not only rift, I'm moving this to cards.tsx